packages/code/src/Code/Code.tsx

```diff
@@ -37,6 +48,10 @@ export function hasMultipleLines(string: string): boolean {
   return string.trim().includes('\n');
 }
 
+function getHorizontalScrollbarHeight(element: HTMLElement): number {
```

Comment:
nit: you could create a `utils` folder inside `code/src` and add this there. `Combobox` is a good example.

Reply:
So the goal in my mind when moving this into a named function wasn't so much to make it reusable, but rather to make it clear what `element.offsetHeight - element.clientHeight` is calculating. I'm not sure if this is a utility method that will be used elsewhere, so I think there's at least some benefit to keeping it close to its usage. Happy to discuss further if this is a well established pattern we follow!

Reply:
I think it's okay that it's not reusable; it's more to avoid bloating the file, but I don't feel too strongly about this.

Reply:
I can see the trade-off here. Given that it's just a one line function created for readability, and is similar with the other util method above it, I don't love the idea of putting it further away from the usage, causing navigation between files to be necessary in order to gain context. I think I'd see benefit in just in-lining it in that case to be more in line with the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))